JobConfig/primary/prolog.fcl

```diff
@@ -33,6 +33,11 @@ Primary: {
       module_type : FindMCPrimary
       SimParticles : "compressDetStepMCs"
     }
+    PBISim : {  
```

Comment:
I think this config should move to digitize/prolog, then  PBISim will get overwritten in mixing/prolog